assets/package.json

```diff
@@ -15,14 +15,18 @@
   "devDependencies": {
     "@babel/core": "^7.0.0",
     "@babel/preset-env": "^7.0.0",
+    "autoprefixer": "^10.2.4",
     "babel-loader": "^8.0.0",
     "copy-webpack-plugin": "^5.1.1",
     "css-loader": "^3.4.2",
-    "sass-loader": "^8.0.2",
-    "node-sass": "^4.13.1",
     "hard-source-webpack-plugin": "^0.13.1",
     "mini-css-extract-plugin": "^0.9.0",
+    "node-sass": "^4.13.1",
     "optimize-css-assets-webpack-plugin": "^5.0.1",
+    "postcss": "^8.2.6",
+    "postcss-loader": "^4.2.0",
+    "sass-loader": "^8.0.2",
```

Comment:
I'm concerned why sass was included in the project when most of what it offers can be achieved using postcss and plugins (postcss-nested, postcss-import, etc.).

Reply:
Tailwind CSS uses scss to build it it's classes @KendyHailstorm

Reply:
It’s not compulsory, I use tailwindcss without any preprocessor like sass, less, stylus and so on. And the reason I don’t use any preprocessor is because most of what they provide can be achieved using a corresponding postcss plugin.

Reply:
I don’t know if I’m allowed to make modification to this same branch/PR. I can make the modifications needed